web/php/Wikitext/FTML/FtmlBackend.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
The Wikidot namespace is intended for legacy code. Any new code should go in one of the laravel-provided folders, likely the `app` folder as everything there inherits the `Wikijump` namespace.

Reply:
Ah alright, I will adjust accordingly.